libqtile/bar.py

```diff
@@ -189,11 +189,13 @@ def __init__(self, widgets, size, **config):
 
         self.queued_draws = 0
         self.future = None
+        self._borders_drawn = False
 
     def _configure(self, qtile, screen):
         # We only want to adjust margin sizes once unless there's a new strut
         if not self._configured or self._add_strut:
```

Comment:
Would `self._configured` work or is the logic a bit different?

Reply:
Different. `_configured` is set after `self._configure` completes but we do need to draw the borders once after that point.